google-cloud-tasks/synth.py

```diff
@@ -37,20 +37,36 @@
 s.copy(v2beta2_library / '.yardopts')
 s.copy(v2beta2_library / 'google-cloud-tasks.gemspec', merge=ruby.merge_gemspec)
```

Comment:
Please remove all the files that are being overwritten by v2beta3. i.e. README.md, LICENSE, .gitignore, .yardops, and the gemspec.

Indeed, it's better to be precise about which files are being copied, without overlaps, now that we're generating more than one library version. For example, instead of blindly copying all of `lib`, instead, from v2beta2, copy:
* `lib/google/cloud/tasks/v2beta2`
* `lib/google/cloud/tasks/v2beta2.rb`

and from v2beta3, copy:
* `lib/google/cloud/tasks/v2beta3`
* `lib/google/cloud/tasks/v2beta3.rb`
* `lib/google/cloud/tasks.rb`

And similar for under `test`.